### PR TITLE
feat(lambda): nodejs14.x runtime

### PR DESCRIPTION
**Pull request by trivikr**
Blog post: [Node.js 14.x runtime now available in AWS Lambda](https://aws.amazon.com/blogs/compute/node-js-14-x-runtime-now-available-in-aws-lambda/)
Developer Guide: [Node.js lambda runtimes](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html)

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/aws-lambda/lib/runtime.ts [hunk 1]
```diff
@@ -78,6 +78,11 @@ export class Runtime {
    */
   public static readonly NODEJS_12_X = new Runtime('nodejs12.x', RuntimeFamily.NODEJS, { supportsInlineCode: true });
 
+  /**
+   * The NodeJS 14.x runtime (nodejs14.x)
+   */
+  public static readonly NODEJS_14_X = new Runtime('nodejs14.x', RuntimeFamily.NODEJS, { supportsInlineCode: true });
```
Comment (coderbyheart) [inline packages/@aws-cdk/aws-lambda/lib/runtime.ts:84]:
```suggestion
  public static readonly NODEJS_14_X = new Runtime('nodejs14.x', RuntimeFamily.NODEJS, { supportsInlineCode: false });
```

I have tested this, and as of now the runtime does not support inline code in `eu-central-1`

> 8/17 | 8:24:49 AM | CREATE_FAILED        | AWS::Lambda::Function       | Lambda (LambdaD247545B) ZipFile can only be used when Runtime is set to either of nodejs8.10, nodejs10.x, nodejs12.x, python2.7, python3.6, python3.7, python3.8.

**Review by coderbyheart — CHANGES_REQUESTED** [review]

**Review by nija-at — DISMISSED** [review]
Thanks for the PR.

Please address the comment above

**Comment by trivikr** [comment]
> Please address the comment above

Addressed :+1:

**Review by nija-at — APPROVED** [review]

**Comment by lielran** [comment]
Thanks for the quick PR and merge. happy to see cdk stay up to date with upgrades of aws services.

**Pull request merged**